minecraft/src/main/java/net/fabricmc/loader/impl/game/minecraft/McLibrary.java

```diff
@@ -34,10 +34,11 @@ enum McLibrary implements LibraryType {
 	LOG4J_PLUGIN_3("net/minecrell/terminalconsole/util/LoggerNamePatternSelector.class"), // in terminalconsoleappender, used by loom's log4j config
 	GSON("com/google/gson/TypeAdapter.class"), // used by log4j plugins
 	SLF4J_API("org/slf4j/Logger.class"),
-	SLF4J_CORE("META-INF/services/org.slf4j.spi.SLF4JServiceProvider");
+	SLF4J_CORE("META-INF/services/org.slf4j.spi.SLF4JServiceProvider"),
+	LOG4J_SLF4J_IMPL("org/apache/logging/slf4j/SLF4JServiceProvider.class");
```

Comment:
Shouldn't the `SLF4J_CORE` definition above work already since log4j uses the same service provider file? (Assuming SLF4J v2 since the class doesn't exist in the SLF4J v1 adapter)

Reply:
Yes, good spot. This was a left over from when I had the classpath all messed up. Now works without this.